src/test/groovy/DatabaseDrivenSpec.groovy

```diff
@@ -17,21 +17,23 @@
 import groovy.sql.Sql
 import spock.lang.Shared
 import spock.lang.Specification
+import spock.lang.Unroll
 
 class DatabaseDrivenSpec extends Specification {
   @Shared sql = Sql.newInstance("jdbc:h2:mem:", "org.h2.Driver")
-  
+
   // insert data (usually the database would already contain the data)
   def setupSpec() {
     sql.execute("create table maxdata (id int primary key, a int, b int, c int)")
-    sql.execute("insert into maxdata values (1, 3, 7, 7), (2, 5, 4, 5), (3, 9, 9, 9)")
+    sql.execute("insert into maxdata values (1, 3, 7, 7), (2, 5, 4, 5), (3, 9, 9, 9), (4, 2, -3, 2)")
   }
 
-  def "maximum of two numbers"() {
+  @Unroll
```

Comment:
Unroll is implicit in 2.0

Reply:
Yes, and it is a breaking change. I thought that in a project made for beginners it would make sense to do things explicitly, especially given the fact that I changed the method name to use variable name expansion.

Reply:
The only reason that `@Unroll` was not deprecated, is that you can still use it to give a different naming format. And that you can switch to old behavior via configuration. That said, I don't think that beginners should learn redundant stuff.